dpnp/backend/kernels/dpnp_krnl_mathematical.cpp

```diff
@@ -411,6 +535,87 @@ void func_map_init_mathematical(func_map_t& fmap)
+    fmap[DPNPFuncName::DPNP_FN_MULTIPLY][eft_LNG][eft_BLN] = {eft_LNG, (void*)dpnp_multiply_c<long, long, bool>};
+    fmap[DPNPFuncName::DPNP_FN_MULTIPLY][eft_LNG][eft_INT] = {eft_LNG, (void*)dpnp_multiply_c<long, long, int>};
+    fmap[DPNPFuncName::DPNP_FN_MULTIPLY][eft_LNG][eft_LNG] = {eft_LNG, (void*)dpnp_multiply_c<long, long, long>};
+    fmap[DPNPFuncName::DPNP_FN_MULTIPLY][eft_LNG][eft_FLT] = {eft_DBL, (void*)dpnp_multiply_c<double, long, float>};
+    fmap[DPNPFuncName::DPNP_FN_MULTIPLY][eft_LNG][eft_DBL] = {eft_DBL, (void*)dpnp_multiply_c<double, long, double>};
+    fmap[DPNPFuncName::DPNP_FN_MULTIPLY][eft_FLT][eft_BLN] = {eft_FLT, (void*)dpnp_multiply_c<float, float, bool>};
+    fmap[DPNPFuncName::DPNP_FN_MULTIPLY][eft_FLT][eft_INT] = {eft_DBL, (void*)dpnp_multiply_c<double, float, int>};
+    fmap[DPNPFuncName::DPNP_FN_MULTIPLY][eft_FLT][eft_LNG] = {eft_DBL, (void*)dpnp_multiply_c<double, float, long>};
+    fmap[DPNPFuncName::DPNP_FN_MULTIPLY][eft_FLT][eft_FLT] = {eft_FLT, (void*)dpnp_multiply_c<float, float, float>};
+    fmap[DPNPFuncName::DPNP_FN_MULTIPLY][eft_FLT][eft_DBL] = {eft_DBL, (void*)dpnp_multiply_c<double, float, double>};
+    fmap[DPNPFuncName::DPNP_FN_MULTIPLY][eft_DBL][eft_BLN] = {eft_DBL, (void*)dpnp_multiply_c<double, double, bool>};
+    fmap[DPNPFuncName::DPNP_FN_MULTIPLY][eft_DBL][eft_INT] = {eft_DBL, (void*)dpnp_multiply_c<double, double, int>};
+    fmap[DPNPFuncName::DPNP_FN_MULTIPLY][eft_DBL][eft_LNG] = {eft_DBL, (void*)dpnp_multiply_c<double, double, long>};
+    fmap[DPNPFuncName::DPNP_FN_MULTIPLY][eft_DBL][eft_FLT] = {eft_DBL, (void*)dpnp_multiply_c<double, double, float>};
+    fmap[DPNPFuncName::DPNP_FN_MULTIPLY][eft_DBL][eft_DBL] = {eft_DBL, (void*)dpnp_multiply_c<double, double, double>};
+    fmap[DPNPFuncName::DPNP_FN_MULTIPLY][eft_C128][eft_C128] = {
+        eft_C128, (void*)dpnp_multiply_c<std::complex<double>, std::complex<double>, std::complex<double>>};
+
+    fmap[DPNPFuncName::DPNP_FN_MULTIPLY_ARRAY_SCALAR][eft_BLN][eft_BLN] = {
+        eft_BLN, (void*)dpnp_multiply_c<bool, bool, bool>};
+    fmap[DPNPFuncName::DPNP_FN_MULTIPLY_ARRAY_SCALAR][eft_BLN][eft_INT] = {
+        eft_LNG, (void*)dpnp_multiply_c<long, bool, int>};
+    fmap[DPNPFuncName::DPNP_FN_MULTIPLY_ARRAY_SCALAR][eft_BLN][eft_LNG] = {
+        eft_LNG, (void*)dpnp_multiply_c<long, bool, long>};
+    fmap[DPNPFuncName::DPNP_FN_MULTIPLY_ARRAY_SCALAR][eft_BLN][eft_FLT] = {
+        eft_DBL, (void*)dpnp_multiply_c<double, bool, float>};
+    fmap[DPNPFuncName::DPNP_FN_MULTIPLY_ARRAY_SCALAR][eft_BLN][eft_DBL] = {
+        eft_DBL, (void*)dpnp_multiply_c<double, bool, double>};
+    fmap[DPNPFuncName::DPNP_FN_MULTIPLY_ARRAY_SCALAR][eft_INT][eft_BLN] = {
+        eft_INT, (void*)dpnp_multiply_c<int, int, bool>};
+    fmap[DPNPFuncName::DPNP_FN_MULTIPLY_ARRAY_SCALAR][eft_INT][eft_INT] = {
+        eft_INT, (void*)dpnp_multiply_c<int, int, int>};
+    fmap[DPNPFuncName::DPNP_FN_MULTIPLY_ARRAY_SCALAR][eft_INT][eft_LNG] = {
```

Comment:
Is it really eft_INT * eft_LNG = eft_INT ?
It is not a request for changes. It is just a note. I think we need to find better solution (I hope we can).

Reply:
Yes, it is.
```python
import numpy
a = numpy.array([1, 2, 3], dtype=numpy.int32)
b = numpy.int64(5).item()  # like b = 5
c = numpy.multiply(a, b)
print(a.dtype, type(b), c.dtype)  # int32 <class 'int'> int32
```
Python `int` means `numpy.int64` (`long`).